### PR TITLE
[TACHYON-1408] Fix flaky lineage integration test

### PR DESCRIPTION
**Pull request by aaudiber**
https://tachyon.atlassian.net/browse/TACHYON-1408

### CODE REVIEW

**Comment by yupeng9** [comment]
Thanks for the fix! Since you're on this, could yo also fix FileOutStreamAsyncWriteIntegrationTest.asyncWriteTest? It has the same problem.

File: integration-tests/src/test/java/tachyon/master/lineage/LineageMasterIntegrationTest.java [hunk 1]
```diff
@@ -160,6 +160,16 @@ public void lineageCompleteAndAsyncPersistTest() throws Exception {
     }
   }
 
+  private void waitForFilePersisted(long fileId) {
```
Comment (apc999) [inline integration-tests/src/test/java/tachyon/master/lineage/LineageMasterIntegrationTest.java]: I am concerned with the way we are using Whitebox in here. It really breaks the encapsulation and requires a lot internal knowledge, meanwhile easy to break once we change the class to be tested.

Can we make a `MockFileSystemWorker` to have this method?

Comment (aaudiber) [inline integration-tests/src/test/java/tachyon/master/lineage/LineageMasterIntegrationTest.java]: This is an integration test, so we shouldn't be mocking something as important as the file system worker. I will create PrivateAccess harnesses similar to https://github.com/amplab/tachyon/blob/master/servers/src/test/java/tachyon/master/file/meta/TtlBucketPrivateAccess.java

Comment (apc999) [inline integration-tests/src/test/java/tachyon/master/lineage/LineageMasterIntegrationTest.java]: sgtm. thanks

**Comment by apc999** [comment]
+1 LGTM

**Comment by yupeng9** [comment]
LGTM

**Pull request merged**